pallets/orml-tokens-extension/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+authors = ["Pendulum Chain"]
+edition = "2021"
+name = "orml-tokens-extension"
```

Comment:
Maybe we should give it a more descriptive name so that it also indicates the feature we are extending it with. How about we call it `orml-tokens-management-extension`?